src/Language/PureScript/Errors.hs

```diff
@@ -639,29 +642,31 @@ prettyPrintSingleError full level e = prettyPrintErrorMessage . positionHintsFir
             , line "Consider moving the instance, if possible, or using a newtype wrapper."
             ]
     renderSimpleErrorMessage InvalidNewtype =
-      line "Newtypes must define a single constructor with a single argument"
+      line "Newtypes must define a single constructor with a single argument."
```

Comment:
Do we need to include the name of the newtype here, or is that obvious in context?